src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -26,7 +26,7 @@ interface Config extends LovelaceConfig {
 
 class HuiEntityButtonCard extends HassLocalizeLitMixin(LitElement)
   implements LovelaceCard {
-  protected hass?: HomeAssistant;
+  public hass?: HomeAssistant;
```

Comment:
we could even consider removing this, as it's already defined in the interface.

Reply:
I thought so too, but the linter didn't.

`Property 'hass' does not exist on type HuiEntityButtonCard`

Reply:
🤷‍♂️